csxtools/lmfit_models.py

```diff
@@ -0,0 +1,200 @@
+import numpy as np

```

Comment:
@vivekthampy @tacaswell I am happy for this to be in `csxtools` if it is not in `scikit-beam` however they absolutely need to have docstrings and comply with `pep8`. Eventually they can migrate into `scikit-beam`. I wanted to check with @licode that we weren't duplicating effort.